### DISCUSSION TITLE
Card title background

### DISCUSSION THREAD

**Discussion by phillip-kruger** [kor-ui/kor#122] [Q&A] [answered]
Hi all. First - Kor is awesome ! I really love how easy I could get started. I have been playing with a few WebComponent UI libraries, and this is my favourite one by far.

I want to create a card (without a photo) where the title bar at the top has a different color. Is that possible ?
(Sorry if this is a dumb question, I am more backend dev than frontend :) )

**Comment by eduferfer** [comment, marked as answer]
Hi, good to know you found Kor easy to use :)

For changing the style of text in the shadow-dom, I'd recommend doing as follows:

```scss
kor-card {
   // this changes the font family, size and style
    --header-1: italic 20px/24px serif;
   // this changes the text color
    --text-1: rgba(200, 50, 100, .90);
}
```

<img width="358" alt="image" src="https://user-images.githubusercontent.com/27731740/185210202-262e7ef3-0e43-49b4-9fd7-db3a7862bc1f.png">

**Reply by phillip-kruger** [reply]
Hi, thanks for the answer !

Is there any way to change the background color of just the header of the card ?

**Reply by eduferfer** [reply]
Hi @phillip-kruger I'm sorry for misunderstanding your question, I thought you meant the text color.

Changing the title background is currently not supported, but I'm thinking of opening up some of these configurations in all components through css variables, such as:

```scss
kor-card {
  --title-bg-color: #FFBBAA;
}
```

Would that help?

**Reply by phillip-kruger** [reply]
Yes for sure ! Thanks
